### PR TITLE
Minor tweaks to Blink docs

### PR DESCRIPTION
**Pull request by fronzbot**
**Description:**
There were some additional comments from @MartinHjelmare around the same time my last PR #6427  got merged, so I wanted to address them.

- Added simple config example and moved complex example lower in doc
- Fixed typo (motion_enabld instead of motion_enabled)
- Changed int to integer per #6452 

## Checklist:
- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

### CODE REVIEW

**Review by fabaff — APPROVED** [review]
Thanks :bird:

File: source/_components/blink.markdown [hunk 1]
```diff
@@ -77,12 +66,33 @@ sensors:
...
 
-**Note:** Each camera reports two different states, one as `sensor.blink_<camera_name>_status` and the other as `binary_sensor.blink_<camera_name>_motion_enabled`.  The `moition_enabld` property reports if the `camera` is ready to detect motion *regardless if the system is actually armed**.  The `status` property is more descriptive, and can be one of the following states:
+**Note:** Each camera reports two different states, one as `sensor.blink_<camera_name>_status` and the other as `binary_sensor.blink_<camera_name>_motion_enabled`.  The `moition_enabled` property reports if the `camera` is ready to detect motion *regardless if the system is actually armed**.  The `status` property is more descriptive, and can be one of the following states:
```
Comment (MartinHjelmare) [inline source/_components/blink.markdown:69]: moition_enabled -> motion_enabled

**Review by MartinHjelmare — COMMENTED** [review]
Thanks @fronzbot!

I think another typo slipped in.

**Comment by frenck** [comment]
@MartinHjelmare So you've hit the edit button an fixed it right away?

**Comment by MartinHjelmare** [comment]
No.

**Comment by frenck** [comment]
@MartinHjelmare Here you go: #6480

**Pull request merged**